article/RNH_AR_EG_SSTJ_EP_MMC-2016.md

```diff
@@ -88,6 +89,17 @@ science, and engineering Scipy (http://scipy.org/).
 
 # Methods
 
+The free water elimination DTI (fwDTI) model describes the measured diffusion-weighted signal $s_i$
+with a simple bi-exponential expansion of DTI:
+
+$$ s_i = s_0 \left [ f \exp\left ( -b D_{iso} \right ) + 
+      (1-f) \exp\left ( -b g_i D g_i \right )\right ] $$ {#eq:1}
+
+where $s_0$ is the signal when no diffusion sensitization gradient is applied, $f$ is the volume fraction
+of free water contamination, $D_iso$ is the free water isotropic diffusion coefficient which is set to $3.0 \times 10^{-3}  mm^{2}.s^{-1}$,
+$D$ is the diffusion tensor of the tissue, $b$ is a constant quantifying the applied diffusion-weighting intensity, and $g_i$
```

Comment:
I wouldn't call $b$ a "constant". How about "$b$ summarizes parameters of the measurement that affect diffusion-weighting intensity"